src/components/Pressable/Pressable.tsx

```diff
@@ -94,8 +94,8 @@ export default function Pressable(props: PressableProps) {
   const pressDelayTimeoutRef = useRef<number | null>(null);
   const pressInHandler = useCallback((event: GestureTouchEvent) => {
     props.onPressIn?.(adaptTouchEvent(event));
-    isPressCallbackEnabled.current = true;
     setPressedState(true);
+    isPressCallbackEnabled.current = true;
```

Comment:
No, It's a visual improvement, bundling together the 2 previously unnecessarily separated refs.

Reply:
Okay, for a second I thought there might be something going on with `asynchronous` nature of `setState`.

Reply:
Thankfully nothing of that sort 😅 

But now that I look at it, I think that could actually be an issue, as this function IS executed asynchronously, and so I'm not sure if this `setState` actually waits for the rest of the code to be executed...

I'll move it to the bottom of this function just in case, as then there would be no cause for concern.